ports/edflib/CMakeLists.txt

```
@@ -0,0 +1,70 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(EDFlib VERSION 1.21 LANGUAGES C)
+
+option(BUILD_TOOLS "Build EDFlib tools." OFF)
+
+include(GNUInstallDirs)
+
+set(sources edflib.h edflib.c)
+
+add_library(EDFlib ${sources})
+
+if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
+    target_compile_definitions(EDFlib PRIVATE _LARGEFILE64_SOURCE _LARGEFILE_SOURCE)

```

```suggestion
    target_compile_definitions(EDFlib PRIVATE _LARGEFILE64_SOURCE _LARGEFILE_SOURCE)
    target_compile_options(EDFlib PRIVATE -Wall -Wextra -Wshadow -Wformat-nonliteral -Wformat-security)
```
See https://gitlab.com/Teuniz/EDFlib/-/blob/master/Makefile#L2